modules/default-page/index.js

```diff
@@ -17,13 +17,16 @@ module.exports = {
       basics: {
         label: 'Basics',
         fields: [
-          'title',
-          'visibility'
+          'title'
```

Comment:
At this point we should just put main in the first tab with title.

Reply:
i always thought this was more an example of schema division .. it never felt like it made a ton of sense otherwise